tests/NLog.UnitTests/LogToDiagnosticsTests.cs

```diff
@@ -0,0 +1,260 @@
+// 
```

Comment:
Please move these tests to `LogToDiagnosticsTests.cs`. Otherwise, it's difficult to find them.